doc/api/pymongo/event_loggers.rst

```diff
@@ -0,0 +1,6 @@
+:mod:`event_loggers` -- Example loggers
+===========================================
+
+.. automodule:: pymongo.event_loggers
+   :synopsis: A collection of simple listeners for monitoring driver events.
+   :members:
```

Comment:
I think there needs to be some whitespace here.

Reply:
Done.